### PR TITLE
Update yug.json

### PR DESCRIPTION
**Pull request by ReallyYug**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [ ] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/yug.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
         "username": "YUG38",
-        "email": "4yugpatel123@gmail.com"
+        "email": "4yugpatel123@gmail.com",
+        "discord": "name_yug"
     },
     "record": {
-        "A": ["217.174.245.249"],
-        "MX": ["hosts.is-a.dev"],
-        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all"
+        "CNAME": "YUG38.github.io"
```
Comment (DEV-DIBSTER) [inline domains/yug.json:10]: Redirects back to this domain for some reason. Invalid changes. No content is being displayed.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/yug.json (hunk 1 above)
Comment (YUG38) [reply domains/yug.json:10]: how to solve?

Comment (DEV-DIBSTER) [reply domains/yug.json:10]: You have to fix your code on your github repository. URL should load a site, not redirect somewhere else in a endless loop.

Comment (YUG38) [reply domains/yug.json:10]: the website code is fine idk why it doesn't work when i try to use YUG38.github.io it redirects to yug.is-a.dev and if i try to use yug.is-a.dev then it won't load saying site not found

Comment (YUG38) [reply domains/yug.json:10]: Help pls?

**Review by wdhdev — APPROVED** [review]

**Pull request merged**